Automattic/wp-calypso
# Add boost feature flag

Add a feature flag for Boost features on the agency pro dashboard



Related to #

## Proposed Changes

* Add a feature flag for Boost features on the agency pro dashboard

## Testing Instructions



* Use the Horizon link below to ensure that the Boost column is no longer available in the agency pro dashboard and the card in the expandable block his also hidden.
* Ensure that the values in the config files for the existing expandable block and new boost value match for the environment and config file.

## Pre-merge Checklist
